.github/pull_request_template.md

```diff
@@ -1,25 +1,14 @@
 <!-- if needed please write above the given line -->
+### Thanks for your PR! Please fill out the following list :)
+
 ---
-Release Notes:
+- [ ] I put the ticket in review
+- [ ] Sonarcloud link : \<add the link here>
+- [ ] No documentation update needed
+- [ ] Link to PR with documentation update: \<add the link here>
+- [ ] No Release Notes needed
+- [ ] Release Notes :
 <!-- Please write your release notes between ```-->
 ```
 
-```
-<!-- please don't delete/modify the checklist --> 
-### For the submitter:
-- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
-- [ ] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
-- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
-- [ ] After integration of the PR, I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
-- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
-- [ ] I put the ticket in review
-- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken
```

Comment:
Don't we want to do this anymore?

Reply:
It would be good to do this. However, in my opinion, writing it into the PR list doesn't help as a reminder, because it's done after integration. Right now, we mostly never do this.